tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -127,3 +131,33 @@ def test_query_sheet_linked_bq_table(self):
             expected_dict,
             columns=["site_name", "hpo_id", "site_point_of_contact"])
         pd.testing.assert_frame_equal(actual_df, expected_df)
+
```

Comment:
FYI, while not your addition, I was wondering what would happen to the tests when run using `application_default_crednetials`.  So I set up my credentials  and `unset GOOGLE_APPLICATION_CREDENTIALS`.  `test_query_sheet_linked_bq_table` failed to get a client object.